infra/index.ts

```diff
@@ -2,15 +2,28 @@ import * as pulumi from "@pulumi/pulumi";
 import * as k8s from "@pulumi/kubernetes";
 import * as awsx from "@pulumi/awsx";
 
+// Get the name of the current stack
+const currentStackName = pulumi.getStack();
+
 // Grab some configuration values
 const config = new pulumi.Config();
-const k8sOrgName = config.require("k8sOrgName");
-const k8sProjName = config.require("k8sProjName");
-const k8sStackName = config.require("k8sStackName");
+const platformOrgName = config.get("platformOrgName") || "zephyr";
```

Comment:
Similar remark as made in zephyr-k8s PR: I prefer the `getOrganization` call here.

Reply:
Fixed via d3c2ea3, thanks for the suggestion